From 1654b7a555ffe7b8d7cc489e75528baecd960635 Mon Sep 17 00:00:00 2001
From: Fang Hui <b31070@freescale.com>
Date: Tue, 4 Jun 2013 09:16:10 +0800
Subject: [PATCH 203/310] ENGR00265228 MX6x: Add audio accessory support for
 sabreaAI and hdmidongle

Add udb config in audio_policy.conf

Signed-off-by: Fang Hui <b31070@freescale.com>
---
 hdmidongle_6dq/audio_policy.conf | 16 ++++++++++++++++
 sabreauto_6q/audio_policy.conf   | 16 ++++++++++++++++
 2 files changed, 32 insertions(+)

diff --git a/hdmidongle_6dq/audio_policy.conf b/hdmidongle_6dq/audio_policy.conf
index 2fca29c..c709621 100644
--- a/hdmidongle_6dq/audio_policy.conf
+++ b/hdmidongle_6dq/audio_policy.conf
@@ -74,4 +74,20 @@ audio_hw_modules {
       }
     }
   }
+ usb {
+    outputs {
+      usb_accessory {
+        sampling_rates 44100
+        channel_masks AUDIO_CHANNEL_OUT_STEREO
+        formats AUDIO_FORMAT_PCM_16_BIT
+        devices AUDIO_DEVICE_OUT_USB_ACCESSORY
+      }
+      usb_device {
+        sampling_rates 44100
+        channel_masks AUDIO_CHANNEL_OUT_STEREO
+        formats AUDIO_FORMAT_PCM_16_BIT
+        devices AUDIO_DEVICE_OUT_USB_DEVICE
+      }
+    }
+  }
 }
diff --git a/sabreauto_6q/audio_policy.conf b/sabreauto_6q/audio_policy.conf
index 0d74cbd..24bce66 100644
--- a/sabreauto_6q/audio_policy.conf
+++ b/sabreauto_6q/audio_policy.conf
@@ -53,4 +53,20 @@ audio_hw_modules {
       }
     }
   }
+ usb {
+    outputs {
+      usb_accessory {
+        sampling_rates 44100
+        channel_masks AUDIO_CHANNEL_OUT_STEREO
+        formats AUDIO_FORMAT_PCM_16_BIT
+        devices AUDIO_DEVICE_OUT_USB_ACCESSORY
+      }
+      usb_device {
+        sampling_rates 44100
+        channel_masks AUDIO_CHANNEL_OUT_STEREO
+        formats AUDIO_FORMAT_PCM_16_BIT
+        devices AUDIO_DEVICE_OUT_USB_DEVICE
+      }
+    }
+  }
 }
-- 
1.8.0

